spec/integrations.yml

```diff
@@ -7,6 +7,10 @@
   mutation_coverage: false
   mutation_generation: true
   exclude:
+    # TODO: Figure out if there is a better way to handle these
+    - core/matchdata/begin_spec.rb
+    - core/matchdata/end_spec.rb
+    # END TODO
```

Comment:
I don't know if there is a good way to handle this better. I thought there were some invalid regexes in here but I'm not sure if that's actually true. I will look into it further.

NOTE: It was excluded via the expected errors thing we used to have in the original.

Reply:
@mbj let me know if you have any insight on these as well but I will also look into it myself a bit more.

Reply:
I opened an issue asking about this here: https://github.com/ammar/regexp_parser/issues/76

I think we could also just rescue the failed parse [here](https://github.com/mbj/mutant/pull/1166/files#diff-8ce8573f1db697fa4d898658487e219a8095e03afd86f897977cdc1a0b8ba26aR18) if we want to and I think we can avoid the exclude.

Reply:
I have added [this](https://github.com/mbj/mutant/pull/1166/commits/147e608a05bca79c3362bc6dae552cd8bcabf947) to fix this for now. @mbj WDYT?

Reply:
this is fine by me.

Reply:
I have squashed it into the primary commit.